src/redux/actions/books.js

```diff
@@ -0,0 +1,45 @@
+// constants
+export const ADD_BOOK = 'ADD_BOOK';
+export const REMOVE_BOOK = 'REMOVE_BOOK';
+export const TOGGLE_BOOK = 'TOGGLE_BOOK';
+
+// action creators
+function addBook(book) {
+  return {
+    type: ADD_BOOK,
+    book,
+  };
+}
+
+function removeBook(id) {
+  return {
+    type: REMOVE_BOOK,
+    id,
+  };
+}
```

Comment:
At the moment, the `remove book` functionality is not working as expected. No action is triggered when I click `remove`. Kindly look into this.

Reply:
It is not meant to be working yet according to the requirements. it will work in another milestone.